src/recipe/Baked.Recipe.Service.Application/Runtime/Configuration/Settings.cs

```diff
@@ -1,7 +1,6 @@
-﻿using Baked.Configuration;
-using Microsoft.Extensions.Configuration;
+﻿using Microsoft.Extensions.Configuration;
 
-namespace Baked;
+namespace Baked.Runtime.Configuration;
```

Comment:
this was intentional. leave this one under `Baked` this is used directly from `Program.cs` where we should require no using statements.